eth/vm/logic/stack.py

```diff
@@ -11,9 +11,9 @@ def pop(computation: BaseComputation) -> None:
 
 def push_XX(computation: BaseComputation, size: int) -> None:
     raw_value = computation.code.read(size)
-
-    if not raw_value.strip(b'\x00'):
-        computation.stack_push(0)
+    raw_len = len(raw_value)
+    if raw_len == size:
```

Comment:
Calling `len()` is shorter than calling an `ljust()` that takes no action (according to profiling), so we try to avoid the `ljust()`. (In fact, the `import_block()` calls in the four mainnet blocks that I'm testing *never* call `ljust()`, over the course of 154k push opcodes).